package/setup.py

```diff
@@ -464,23 +534,28 @@ def dynamic_author_list():
           packages=find_packages(),
           package_dir={'MDAnalysis': 'MDAnalysis'},
           ext_package='MDAnalysis',
-          ext_modules=exts,
+          ext_modules=extensions(config, use_cython),
           classifiers=CLASSIFIERS,
-          cmdclass=cmdclass,
-          requires=['numpy (>=1.5.0)', 'biopython',
-                    'networkx (>=1.0)', 'GridDataFormats (>=0.3.2)'],
+          distclass=MDADistribution,
+          mda_use_cython=use_cython,
+          requires=['numpy (>=1.5.0)',
+                    'biopython',
+                    'networkx (>=1.0)',
+                    'GridDataFormats (>=0.3.2)',
+                    'six (>=1.4.0)',
+                    ] + require_cython[:1],
           # all standard requirements are available through PyPi and
           # typically can be installed without difficulties through setuptools
-          setup_requires=[
+          mda_build_requires=[
               'numpy>=1.5.0',
-          ],
+              ] + require_cython[1:],
```

Comment:
This is indirect like this because when doing release installs we don't want to burden the user with cython (we're distributing the compiled cython stuff anyway).